servicetalk-http-api/src/main/java/io/servicetalk/http/api/ProxyConnectException.java

```diff
@@ -13,39 +13,36 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.servicetalk.http.netty;
-
-import io.servicetalk.transport.api.RetryableException;
+package io.servicetalk.http.api;
```

Comment:
This class was not released yet (introduced in #2711). Safe to move without going through deprecation process